pkg/provider/branch/trunk/trunk.go

```diff
@@ -188,7 +188,7 @@ func (t *trunkENI) InitTrunk(instance ec2.EC2Instance, podList []v1.Pod) error {
 		}
 
 		trunk, err := t.ec2ApiHelper.CreateAndAttachNetworkInterface(&instanceID, aws.String(t.instance.SubnetID()),
-			t.instance.InstanceSecurityGroup(), nil, &freeIndex, &TrunkEniDescription, &InterfaceTypeTrunk, 0)
+			t.instance.InstanceSecurityGroup(), nil, &freeIndex, &TrunkEniDescription, &InterfaceTypeTrunk, 0, 0)
```

Comment:
Please check my previous comment. I am not certain if we want to add more arguments in the interface function, especially this function is implemented by another feature which doesn't need IP resources passed in.

Reply:
Makes sense, updated!